docs/troubleshooting.md

```diff
@@ -3,30 +3,116 @@
+
+### mount: exec: "fusermount": executable file not found in $PATH`
+
+Unable to mount with the following error `daemonize.Run: readFromProcess: sub-process: Error while mounting gcsfuse: mountWithArgs: mountWithStorageHandle: Mount: mount: exec: "fusermount": executable file not found in $PATH`
+
+Fuse package is not installed. It may throw this error. Run the following command to install fuse <br/> <code> sudo apt-get install fuse </code> for Debian/Ubuntu <br/> <code> sudo yum install fuse </code> for RHEL/CentOs/Rocky <br/>
+
+### ls: reading directory \<mountpath>/\<path>:  Input/output error
+
+Find out if you have any object(s) with name/prefix having `//` in it or starting with `/`, in the mounted GCS bucket (use `gsutil ls gs://<bucket>/<path>` to find out). If yes, move/rename such objects to name/prefix not having `//` or starting with `/` (e.g. for object/prefix `A//B`, use `gsutil -m mv -r gs://<bucket>/A//* gs://<bucket>/A/`), or delete it (use `gsutil -m rm -r A//`). Refer [semantics](semantics.md#unsupported-object-names) for more details.
+
+### Experiencing hang while executing "ls" on a directory containing large number of files/directories.
+
+By default `ls` does listing but sometimes additionally does `stat` for each list entry. Additional stat on each entry slows down the entire `ls` operation and may look like hang, but in reality is iterating one by one and slow. There are two ways to overcome this: <ul><li>**Get rid of `stat`:** Execute without coloring `ls --color=never` or `\ls` to remove the stat part of `ls`.</li> <li>**Improve the `stat` lookup time:** increase the metadata ([stat](https://cloud.google.com/storage/docs/gcsfuse-cache#stat-cache-overview) + [type](https://cloud.google.com/storage/docs/gcsfuse-cache#type-cache-overview)) cache ttl/capacity which is populated as part of listing and makes the `stat` lookup faster. Important to note here, `ls` will be faster from the first execution itself as cache is populated in listing phase, leading to quicker stat lookup for each entry.</li></ul>
+
+### GCSFuse Errors on Google Cloud Console Metrics Dashboard: "CANCELLED ReadObject" & "NOT_FOUND GetObjectMetadata"
+
+Both these errors are expected and part of GCSFuse standard operating procedure. More details [here](https://github.com/GoogleCloudPlatform/gcsfuse/discussions/2300).
+
+### GCSFuse logs showing errors for StatObject NotFoundError 
+
+`StatObject(\"<object_name>") (<time>ms): gcs.NotFoundError: storage: object doesn't exist"`.
+This is an expected error. Please refer to **NOT_FOUND GetObjectMetadata** section [here](https://github.com/GoogleCloudPlatform/gcsfuse/discussions/2300#discussioncomment-10261838).
```

Comment:
nit: Not sure about this but we should probably add a **Fix:** tag before the fix part for easy spotting. This might be helpful in cases, where description is long.

Reply:
This will also be more faithful to the original information in the table.

Reply:
I am assuming the whole section is meant to be fix details so not very sure if we should add a "fix:" . 
Ironically, for this particular issue there is no fixing, its just telling this is as expected.